commander/doc.go

```diff
@@ -0,0 +1,7 @@
+/*Package commander is bassicly a connection to redis which support for multipe chained commands,
```

Comment:
> Package Commander provides a Redis connection which can run multiple Redis commands by chaining. The connection wraps and enhances its Redigo counterpart.

Typo:
bassicly => basically
support for multipe chained commands => supports multipe chained commands